### PR TITLE
fix: add autoscaling to oracle api

### PR DESCRIPTION
**Pull request by craigyu**
Add autoscaling to oracle api, configured the same way as backend service
DEV: Min=1, Max=1
TEST & PROD: Min=3, MAX=5

---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Backend](https://nr-spar-170-backend.apps.silver.devops.gov.bc.ca/)
[Frontend](https://nr-spar-170-frontend.apps.silver.devops.gov.bc.ca/)
[Oracle-API](https://nr-spar-170-oracle-api.apps.silver.devops.gov.bc.ca/)

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-spar/actions/workflows/merge-main.yml)

### CODE REVIEW

**Review by mgaseta — APPROVED** [review]
LGTM! 🚀✔

**Review by DerekRoberts — APPROVED** [review]
Good improvements.  :)

File: .github/workflows/pr-open.yml [hunk 1]
```diff
@@ -143,12 +146,16 @@ jobs:
...
+              -p MIN_REPLICAS=1
+              -p MAX_REPLICAS=1
           - name: oracle-api
             file: oracle-api/openshift.deploy.yml
             overwrite: true
             verification_path: "actuator/health"
-            parameters: -p NR_SPAR_ORACLE_API_VERSION=snapshot-${{ github.event.number }}
+            parameters:
+              -p NR_SPAR_ORACLE_API_VERSION=snapshot-${{ github.event.number }}
+              -p MIN_REPLICAS=1
```
Comment (DerekRoberts) [inline .github/workflows/pr-open.yml:157]: Hahaha!  You sure took that "piss me off" comment to heart.  :D + :P

**Review by DerekRoberts — APPROVED** [review]

File: .github/workflows/pr-open.yml (hunk 1 above)
Comment (craigyu) [reply .github/workflows/pr-open.yml:157]: 😈

**Pull request merged**